agent/tool-scripts/prometheus-metrics

```diff
@@ -126,12 +151,16 @@ elif [[ ! -s "$inventory" ]] || [[ ! -f "$inventory" ]]; then
 	echo "Please check if the inventory file exists, is not empty and the inventory variable is not set to a directory"
 	exit 1
 fi
+## set default values for port if not defined
+if [ -z "$port" ]; then
+        port=8443
+fi
 ## parse openshift inventory file
 cat "$inventory" | awk -F " " '{ print $1 }' | sed -n '/^\[masters\]/,/^$/p' | awk '{if (NR!=1) {print}}' | sed '/^$/d' > /run/pbench/inv_hosts
 case "$mode" in
 	install)
 	which go &>/dev/null
-	if [[ $(echo $?) != 0 ]]; then
+	if [[ $? != 0 ]]; then
 		yum -y install go &>/dev/null
 		mkdir $pbench_install_dir/.go	
 		echo "GOPATH=$pbench_install_dir/.go" >> ~/.bashrc
```

Comment:
@ndokos GOPATH stuff is tracked as a seperate PR - https://github.com/distributed-system-analysis/pbench/pull/610